FBSnapshotTestCase/Categories/UIImage+Snapshot.m

```diff
@@ -43,7 +43,7 @@ + (UIImage *)fb_imageForView:(UIView *)view
   NSAssert1(CGRectGetWidth(bounds), @"Zero width for view %@", view);
   NSAssert1(CGRectGetHeight(bounds), @"Zero height for view %@", view);  
 
-  UIWindow *window = view.window;
```

Comment:
:+1: nice catch. 

Can you add a comment here to explain why we do this? Please also squash the commits once you update the code and I'll merge it in.